tests/antithesis/docker-compose.yml

```diff
@@ -46,4 +46,4 @@ services:
   client:
     image: 'etcd-client:latest'
     container_name: client
-    entrypoint: ["/entrypoint.py"]
+    entrypoint: ["/entrypoint/entrypoint.go"]
```

Comment:
I think you need the entrypoint to point to the binary `/opt/antithesis/test/v1/main/entrypoint`.